src/backend/common/CommandBuffer.cpp

```diff
@@ -201,6 +201,19 @@ namespace backend {
             auto it = mostRecentTextureUsages.find(texture);
             return it != mostRecentTextureUsages.end() && (it->second & usage);
         };
+        auto isTextureTransitionPossible = [&](TextureBase* texture, nxt::TextureUsageBit usage) -> bool {
+            const nxt::TextureUsageBit attachmentUsages =
+                nxt::TextureUsageBit::ColorAttachment |
+                nxt::TextureUsageBit::DepthStencilAttachment;
+            ASSERT(usage != nxt::TextureUsageBit::None && nxt::HasZeroOrOneBits(usage));
+            auto it = mostRecentTextureUsages.find(texture);
```

Comment:
Oh, I forgot to fix this case. I was going to also forbid explicit transitions INTO attachment usages. WDYT?